drug-drug-interaction/cli/drug-drug-interaction-eval.data

```diff
@@ -0,0 +1,27 @@
+predicates:
+    ATCSimilarity/2: closed
+    SideEffectSimilarity/2: closed
+    GOSimilarity/2: closed
+    ligandSimilarity/2: closed
+    chemicalSimilarity/2: closed
+    seqSimilarity/2: closed
+    distSimilarity/2: closed
+    validInteraction/2: closed
+    interacts/2: open
+
+observations:
+   ATCsimilarity: ../data/general-interactions/00/eval/similarity_atc.txt
```

Comment:
The path should be `../data/drug-drug-interaction/general-interactions/00/eval/similarity_atc.txt`.